mmv1/templates/terraform/examples/bigquerydatatransfer_config_scheduled_query.tf.erb

```diff
@@ -25,7 +25,7 @@ resource "google_bigquery_data_transfer_config" "<%= ctx[:primary_resource_id] %
 resource "google_bigquery_dataset" "my_dataset" {
   depends_on = [google_project_iam_member.permissions]
 
-  dataset_id    = "<%= ctx[:vars]['dataset_id'].delete("-") %>"
+  dataset_id    = "<%= ctx[:vars]['dataset_id'] %>"
```

Comment:
what did this do before? I guess it's not needed?

Reply:
It deletes the `-` from the value of dataset_id. Yes, it is not needed.   The value of `dataset_id` is `my_dataset` and doesn't have `-`

https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/products/bigquerydatatransfer/Config.yaml#L48